docs/user_guide/command_line.md

```diff
@@ -456,6 +457,68 @@ The Merlin library provides several commands for setting up and managing your Me
 - *[run workers](#run-workers-merlin-run-workers)*: Start up workers that will execute the tasks that exist on the central server
 - *[stop workers](#stop-workers-merlin-stop-workers)*: Stop existing workers
 
+### Cancel (`merlin cancel`)
+
+The `merlin cancel` command allows you to cancel a running study. A study may have multiple runs executing concurrently on multiple workers being watched by a monitor process to ensure every run finishes. This command will help ensure that every piece of this process is cancelled gracefully.
+
+In other words, the `merlin cancel` command acts as a wrapper around the [`merlin purge`](#purge-merlin-purge) and [`merlin stop-workers`](#stop-workers-merlin-stop-workers) commands. Additionally, any running studies associated with the study being cancelled will be marked as cancelled in the database. This ensures that the `merlin monitor` command will no longer track these studies.
+
+In short, this command will:
```

Comment:
maybe expand the command here? `merlin cancel MY_STUDY` and use `MY_STUDY` in 1. 2. and 3. ?